microshift_updating/microshift-update-options.adoc

```diff
@@ -48,10 +55,19 @@ To understand more about Greenboot, see the following documentation:
 
 [id="microshift-update-options-manual-rpm-updates_{context}"]
 === Manual RPM updates
-You can update {microshift-short} manually on a non-OSTree system such as {op-system-base-full} by downloading and updating the RPMs. To complete this update type, use the subscription manager to access the repository containing the new RPMs. To begin a manual RPM update, use the procedures in the following documentation:
+You can update {microshift-short} manually on a non-OSTree system such as {op-system-base-full} by updating the RPMs.
+
+* To complete this update type, use the subscription manager to enable the repository containing the new RPMs.
+* Ensuring system health and completing additional system backups are manual processes.
+* To begin a manual RPM update, use the procedures in the following documentation:
 
 * xref:../microshift_updating/microshift-update-rpms-manually.adoc#microshift-update-rpms-manually[About updating MicroShift RPMs manually]
 
+[WARNING]
+====
+Keeping versions in a supported configuration of {op-system-bundle} can require updating {microshift-short} and {op-system} at the same time. Ensure that your version of {op-system} is compatible with the version of {microshift-short} you are updating to, especially if you are updating {microshift-short} across two minor versions. Otherwise, you can create an unsupported configuration and break your cluster.
```

Comment:
```suggestion
Keeping versions in a supported configuration of {op-system-bundle} can require updating {microshift-short} and {op-system} at the same time. Ensure that your version of {op-system} is compatible with the version of {microshift-short} that you want to update, especially if you are updating {microshift-short} across two minor versions. Otherwise, you can create an unsupported configuration that breaks your cluster.
```

Reply:
Hmm, what I really mean here is "...you can create an unsupported configuration, break your cluster, or both."

Good note, thanks!